Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -810,19 +814,30 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
     auto trNegID = svertex.getProngID(1);
     int posTableIdx = -1;
     int negTableIdx = -1;
-    auto item = mV0sIndices.find(trPosID);
-    if (item != mV0sIndices.end()) {
+    auto item = mGIDToTableID.find(trPosID);
+    if (item != mGIDToTableID.end()) {
       posTableIdx = item->second;
     }
-    item = mV0sIndices.find(trNegID);
-    if (item != mV0sIndices.end()) {
+    item = mGIDToTableID.find(trNegID);
+    if (item != mGIDToTableID.end()) {
       negTableIdx = item->second;
     }
     v0sCursor(0, posTableIdx, negTableIdx);
```

Comment:
I would produce an exception if posTableIdx or negTableIdx stay -1.

Reply:
Hi @shahor02. Thank you, I have added a check for this https://github.com/AliceO2Group/AliceO2/blob/b4c18b8454c7777da3626bd967b7cb9f8ba514d3/Detectors/AOD/src/AODProducerWorkflowSpec.cxx#L817-L828

Reply:
could you do this also for the cascades?